client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx

```diff
@@ -262,15 +244,18 @@ class TransferOtherUser extends React.Component {
 							value={ this.state.selectedUserId }
 						>
 							{ availableUsers.length ? (
-								availableUsers.map( ( user ) => {
-									const userId = this.getWpcomUserId( user );
-
-									return (
-										<option key={ userId } value={ userId }>
-											{ this.getUserDisplayName( user ) }
-										</option>
-									);
-								} )
+								<Fragment>
+									<option value="">{ translate( '-- Select user --' ) }</option>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( '-- Select User --' )` *ES Score: 8*